rosetta/services/transaction_parser.go

```diff
@@ -79,6 +99,46 @@ func (tp *transactionsParser) createRosettaTxFromUnsignedTx(eTx *data.FullTransa
 	}, true
 }
 
+func (tp *transactionsParser) createRosettaTxUnsignedTxSendFunds(
+	eTx *data.FullTransaction,
+) (*types.Transaction, bool) {
+	return &types.Transaction{
+		TransactionIdentifier: &types.TransactionIdentifier{
+			Hash: eTx.Hash,
+		},
+		Operations: []*types.Operation{
+			{
+				OperationIdentifier: &types.OperationIdentifier{
+					Index: 0,
+				},
+				Type:   opScResult,
+				Status: OpStatusSuccess,
+				Account: &types.AccountIdentifier{
+					Address: eTx.Sender,
+				},
+				Amount: &types.Amount{
+					Value:    "-" + eTx.Value,
```

Comment:
can you confirm this is only called only for SCR from SC to SC with value transfer

for gas refund value should not be decreased on sender.

Reply:
Yes. This is called for SCR that send user funds (value is != 0 and receiver != sender).